### PR TITLE
Updating Data Sources pages

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: datasets/index.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
 ---
 layout: default
 title: Data Sources
-nav_order: 110
+nav_order: 9
```
Comment (beets) [inline datasets/index.md:4]: I believe @shifucun made these very far apart so it's easier to slot in new pages, just like freeway exit numbers.

**Review by beets — APPROVED** [review]
thanks for the update!

File: datasets/index.md (hunk 1 above)
Comment (shifucun) [reply datasets/index.md]: Yep! I wonder how are these auto-generated? We may want to fix there.

Comment (chejennifer) [reply datasets/index.md]: ok, I reverted for now, but yes should probably fix in the auto-generation

**Pull request merged**